src/frequenz/channels/timer.py

```diff
@@ -751,19 +716,11 @@ def _now(self) -> int:
         return _to_microseconds(self._loop.time())
 
     def __str__(self) -> str:
-        """Return a string representation of the timer.
-
-        Returns:
-            The string representation of the timer.
-        """
+        """Return a string representation of this timer."""
         return f"{type(self).__name__}({self.interval})"
 
     def __repr__(self) -> str:
-        """Return a string representation of the timer.
-
-        Returns:
-            The string representation of the timer.
-        """
+        """Return a string representation of this timer."""
```

Comment:
there is not difference between __str__() and __repr__() docstrings
```suggestion
        """Return a string with the internal representation of this timer."""
```